common/beerocks/tlvf/src/src/beerocks_message.cpp

```diff
@@ -137,9 +137,7 @@ bool message_com::send_cmdu(Socket *sd, ieee1905_1::CmduMessageTx &cmdu_tx,
         return false;
     }
 
-    bool swap = !dst_mac.empty();
-
-    if (!cmdu_tx.finalize(swap)) {
+    if (!cmdu_tx.finalize(true)) {
```

Comment:
If the swap is always true, then maybe it is a good idea to remove the swap argument from the function.

Reply:
happens in the followup commit